R/region_to_chunks.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @description Parse a region string into; chromosome, start and end.
 #'
-#' @details To convert region into chunks of chr, start, and end.
+#' @details INTERNAL FUNCTION called by [GAMBLR::calc_mutation_frequency_bin_region], not meant for out-of-package usage.
```

Comment:
I am not sure, does the devtools::check calls this as a dead link here that points to outside of the package?

Reply:
No. A link to a function of another package works in RStudio.

Reply:
BTW, I changed the prefix of all links like this to the respective child repo. For instance, this line here was changed to 
```
#' @details INTERNAL FUNCTION called by [GAMBLR.utils::calc_mutation_frequency_bin_region], not meant for out-of-package usage.
```